homeassistant/components/simplisafe/__init__.py

```diff
@@ -474,6 +475,27 @@ async def async_init(self):
                 _LOGGER.error("Error while fetching initial event: %s", err)
                 self.initial_event_to_use[system.system_id] = {}
 
+            # The initial update doesn't process notifications because it is likely that
+            # they will trigger SIMPLISAFE_NOTIFICATION events before other critical
+            # components – such as automation – are ready. Therefore, we schedule an
+            # update of notifications as soon as HASS is fully started:
+            async def process_notifications_listener(_):
+                """Define a listener to run when `homeassistant_start` fires."""
+
+                async def process_when_running():
+                    """Process notifications when HASS is running."""
+                    while self._hass.state != CoreState.running:
+                        await asyncio.sleep(1)
+                    self._async_process_new_notifications(  # pylint: disable=W0640
+                        system
+                    )
+
+                return self._hass.loop.create_task(process_when_running())
```

Comment:
Please use `asyncio.create_task` instead. Python is trying to get us away from the loop api to a higher level api.